doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -40,7 +52,13 @@
 <a name="3.2.3"></a>
 ### 3.2.3 Container Storage Services
 
-> This shapter should discuss storage services provided by the reference architecture. 
+Containers, when running, will require ephemeral storage on which to run themselves (i.e. storage on which the unpacked container image is stored and executed from). This is ephemeral storage lives and dies with the container and is a directory on the worker node on which the container is running.  Note, this means that the ephemeral storage is mounted locally in the worker node filesystem. The filesystem can also be physically external to the worker node (e.g. iSCSI, NFS, FC) but the container will still reference it as part of the local filesystem.
+
+Additional ephemeral storage can also be attached to a container through the use of Kubernetes Volumes - this can be storage from the worker node filesystem, or it can be external storage that is accessed through the use of a Volume Plugin. Volume Plugins allow the use of a storage protocol (e.g. iSCSI, NFS) or management API (e.g. Cinder) for the attaching and mounting of storage into a Pod. This additional ephemeral storage, that is attached to a container using a Kubernetes Volume, does not live and die with the container but instead follows the lifecycle of the Pod that the container is a part of. This means the ephemeral Volume persists across container restarts, whilst the Pod is still running.
+
+In Kubernetes, Persistent Volumes (PV) are used when data persistence is required. PVs are resources in a Kubernetes Cluster that are consumed by Persistent Volume Claims (PVCs) and have a lifecycle that is independent of any Pod that uses the PV. A Pod will use a PVC as the volume in the Pod spec; a PVC is a request for persistent storage (a PV) by a Pod. By default, PVs and PVCs are manually created and deleted.
+
+Kubernetes also provides an object called Storage Class, which is created by cluster administrators and maps to storage attributes such as quality-of-service, encryption, data resilience, etc. Storage Classes also enable the dynamic provisioning of Persistent Volumes (as opposed to the default manual creation). This can be beneficial for organisations where the administration of storage is performed separately from the administration of Kubernetes-based workloads.
 
 <a name="3.2.4"></a>
```

Comment:
Since you have capitalised "Pod" in your PR changes, can you also modify pods->Pods in Section 3.2.4 (2 usages) for consistency?

Reply:
Fixed